google-cloud/docs/toc.json

```diff
@@ -182,6 +182,43 @@
         }
       ]
     },
+    {
+      "title": "DLP",
+      "type": "google/cloud/dlp",
+      "patterns": ["\/dlp", "google\/api", "google\/protobuf"],
+      "nav": [
+        {
+          "title": "V2",
+          "type": "google/cloud/dlp/v2",
+          "patterns": ["\/dlp\/v2", "google\/api", "google\/protobuf"],
+          "nav": [
+            {
+              "title": "DLPServiceClient",
+              "type": "google/cloud/dlp/v2/dlpserviceclient"
+            },
+            {
+              "title": "Data Types",
+              "type": "google/dlp/v2/datatypes"
+            }
+          ]
+        },
+        {
+          "title": "V2beta1",
+          "type": "google/cloud/dlp/v2beta1",
+          "patterns": ["\/dlp\/v2beta1", "google\/api", "google\/protobuf"],
+          "nav": [
+            {
+              "title": "DLPServiceClient",
+              "type": "google/cloud/dlp/v2beta1/dlpserviceclient"
+            },
+            {
+              "title": "Data Types",
+              "type": "google/dlp/v2beta1/datatypes"
```

Comment:
I think maybe you copied this over before updating the `type` values? Should be `google/privacy/dlp/v2beta1/datatypes`.

Reply:
nm, fixed in subsequent commit.